src/components/MapView/MapView.tsx

```diff
@@ -52,18 +52,88 @@ const FIT_BOUNDS_OPTIONS = {
   padding: 20
 }
 
+type TableRowProps = {
+  name: string
+  value: number
+}
+
+const TableRow = ({ name, value }: TableRowProps) => (
+  <tr>
+    <td>{name}</td>
+    <td style={{ paddingLeft: 15 }}>{value}</td>
+  </tr>
+)
+
+function providerPopup(point: any) {
+  console.log(point)
+  /**
+   * TODO: expose info for all providers with the same address,
+   * not only the first one.
+   */
+  let pointProps = point.features[0].properties
+  return (
+    <div className='popup-container'>
+      <Popup
+        offset={[0, -20]}
+        anchor='bottom'
+        coordinates={point.lngLat} >
+        <table className='popup-table'>
+          <tbody>
+            <TableRow name='Address' value={pointProps.address} />
+            <TableRow name='Specialty' value={pointProps.specialty} />
+            <TableRow name='Lat' value={point.lngLat['lat'].toFixed(4)} />
+            <TableRow name='Long' value={point.lngLat['lng'].toFixed(4)} />
+          </tbody>
+        </table>
+      </Popup>
+    </div>
+  )
+}
+
+function representativePointPopup(point: any) {
+  let pointProps = point.features[0].properties
+  console.log(point)
+  return (
+    <div className='popup-container'>
+      <Popup
+        offset={[0, -20]}
+        anchor='bottom'
+        coordinates={point.lngLat} >
+        <table className='popup-table'>
+          <tbody>
+            <TableRow name='County' value={pointProps.county} />
+            <TableRow name='ZIP' value={pointProps.zip} />
+            <TableRow name='Population' value={pointProps.population} />
+            <TableRow name='Lat' value={point.lngLat['lat'].toFixed(4)} />
+            <TableRow name='Long' value={point.lngLat['lng'].toFixed(4)} />
+          </tbody>
+        </table>
+      </Popup>
+    </div>
+  )
```

Comment:
`providerPopup` and `representativePointPopup` share a a lot of code. Consider DRYing them out into a higher order component instead:

```ts
// components/MapTooltip/MapTooltip.tsx:

type Props = {
  content: React.ComponentClass
  coordinates: [number, number]
}

let MapTooltip: React.StatelessComponent<Props> = ({ content, coordinates }) =>
  <div className='popup-container'>
    <Popup coordinates={latLng} ...>
       { content }
    </Popup>
  </div>
```

```ts
// components/MapView/MapView.tsx:

import { MapTooltip } from '../MapTooltip/MapTooltip'

type RepresentativePointPopupProps = {..}

let RepresentativePointPopup: React.StatelessComponent<RepresentativePointPopupProps> = ({ point }) =>
  <MapTooltip
    coordinates={point.lngLat}
    content={
      <table className='popup-table'>
        ...
      </table>
    }
  />